Makefile

```diff
@@ -35,3 +55,21 @@ unit-test:
 
 build: $(SOURCES)
 	go build $(GOBUILDFLAGS) -ldflags="$(GOLDFLAGS)" -o ./update-service-operator ./
+
+# Generate manifests e.g. CRD, RBAC etc.
+manifests:
+	controller-gen $(CRD_OPTIONS) rbac:roleName=updateservice-operator webhook paths="./..." output:crd:artifacts:config=config/crd/bases
```

Comment:
SDK scaffolding [includes a section on installing `controller-gen`][1], setting up a variable to use here.  Can we follow that approach?  I like having the expected version of `controller-gen` mentioned in version control.

[1]: https://github.com/openshift/cincinnati-operator/blob/0731f08b2fbd3989b4642ff55c6a216e97d4c990/Makefile#L77-L92
[2]: https://github.com/openshift/cincinnati-operator/blob/0731f08b2fbd3989b4642ff55c6a216e97d4c990/Makefile#L55

Reply:
I just simplified it and prefer explicitly doing installs but we can leave it as generated.

Reply:
you can still do manual installs, with [this branch][1] using your already-installed command (regardless of the version it happens to be).  The scaffolded section just tries to install `controller-gen` if you don't have it in your path yet.

[1]: https://github.com/openshift/cincinnati-operator/blob/0731f08b2fbd3989b4642ff55c6a216e97d4c990/Makefile#L91

Reply:
Added installs back for controller-gen and kustomize as originally generated.